phpBB/includes/acp/acp_extensions.php

```diff
@@ -180,19 +180,13 @@ function main($id, $mode)
 					redirect($this->u_action);
 				}
 
-				if (confirm_box(true))
-				{
-					redirect($this->u_action . '&amp;action=enable&amp;ext_name=' . urlencode($ext_name) . '&amp;hash=' . generate_link_hash('enable.' . $ext_name));
-				}
-				else
-				{
-					confirm_box(false, $this->user->lang('EXTENSION_ENABLE_CONFIRM', $md_manager->get_metadata('display-name')), build_hidden_fields(array(
-						'i'			=> $id,
-						'mode'		=> $mode,
-						'action'	=> 'enable_pre',
-						'ext_name'	=> $ext_name,
-					)));
-				}
+				$this->tpl_name = 'acp_ext_enable';
+
+				$this->template->assign_vars(array(
+					'PRE'				=> true,
+					'L_CONFIRM_MESSAGE'	=> $this->user->lang('EXTENSION_ENABLE_CONFIRM', $md_manager->get_metadata('display-name')),
```

Comment:
Please remove the `L_` prefix, as the message is already translated, so the phpBB template engine doesn't try to translate it again.

Reply:
This is done throughout the core. Do we prefer updating the code as you suggest, or should we keep it the way it is for the sake of consistency?

Reply:
Unfortunately I am aware that it is done throughout the core, and unfortunately it is incorrect in all those places. The `L_` prefix was added by the old phpBB Template syntax, which will try to translate these variables. Seeing these varibales are already translated, it would provide unnecessary overhead to try and translate them again. 
I would like to see the 'correct consistency' across the core aswell, but that's easier said than implemented. And while consistency is important, it should not be the reason to not adjust something, otherwise it will become close to impossible to progress and the code (and its code style) will grind to a halt.